cmd/delete/cluster.go

```diff
@@ -46,6 +47,11 @@ func NewClusterCmd() *cobra.Command {
 				return ErrFuryctlFlagNotSet
 			}
 
+			distroLocation, ok := cobrax.Flag[string](cmd, "distro-location").(string)
+			if !ok {
+				return ErrDistroFlagNotSet
```

Comment:
Wrap this error and add which flag is not set, same for all the flags below.
Perhaps there's a way to mark them as mandatory down below so there's no need to do this here?
It's also a bit weird that all these flags are mandatory, eg: dry-run and force should be defaulted and not required